Resources/Prototypes/Entities/items/clothing/hats.yml

```diff
@@ -23,3 +23,20 @@
     sprite: Clothing/chef_hat.rsi
     Slots:
       - helmet
+
+- type: entity
+  parent: HatBase
+  id: HatCaptain
+  name: Captain's Hat
+  description: It's good being the king.
```

Comment:
Just a small nitpick, but I'd change this line to "It's good to be king"

Reply:
oh, yeah. i just copy pasted what it was in ss13. i didn't pay any mind to it.